tuxemon/formula.py

```diff
@@ -206,6 +206,30 @@ def simple_overfeed(
     return speed
 
 
+def simple_grabbed(monster: Monster) -> None:
+    for move in monster.moves:
+        if move.range.ranged:
+            move.potency = move.potency * 0.5
+            move.power = move.power * 0.5
```

Comment:
This looks too permanent

Reply:
Yeah, it looks, because at the end we are dealing with parameters different from speed, etc. I tested, and after the battle, the values return as before. I was thinking about a set_stat for techniques where the focus was power and potency.